issue99_test.go

```diff
@@ -0,0 +1,78 @@
+package zmq4
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+)
+
+var (
+	wg     sync.WaitGroup
+	outMsg Msg
+	inMsg  Msg
+)
+
+func requester(t *testing.T) {
+
+	req := NewReq(context.Background())
+	defer req.Close()
+
+	err := req.Dial("tcp://localhost:5559")
```

Comment:
instead of hardcoding `5559` as a port, please use the `EndPoint` function from https://github.com/go-zeromq/zmq4/blob/master/zmq4_test.go#L29.

the port would be fetched inside `TestIssue99` and closed around inside the two closures for `requester` and `responder`.

Reply:
Oh nice, will change that.